collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java

```diff
@@ -142,9 +142,9 @@ public void playVideo() {
 
             Intent i = new Intent("android.intent.action.VIEW");
             i.setDataAndType(Uri.fromFile(videoFile), "video/*");
-            try {
+            if (i.resolveActivity(getContext().getPackageManager()) != null) {
```

Comment:
@lognaturel the changes in the medialayout file are in the main repo as well i guess i had deleted those changes you are might be comparing this branch with your repo.